Tests/WeaverTests/RealmWeaver.Tests/WeaverTests.cs

```diff
@@ -135,42 +93,50 @@ private void WeavePropertyChanged(string targetAssemblyPath)
         private readonly List<string> _warnings = new List<string>();
         private readonly List<string> _errors = new List<string>();
 
-        private readonly WeaverOptions _weaverOptions;
+        private readonly WeaverOptions? _weaverOptions;
 
-        public WeaverTests(WeaverOptions weaverOptions)
+        public Tests(WeaverOptions weaverOptions)
         {
             _weaverOptions = weaverOptions;
         }
 
         [OneTimeSetUp]
         public void FixtureSetup()
         {
-            var assemblyDir = GetAssemblyDir();
-            TryCleanupOldAssemblies(assemblyDir);
+            _sourceAssemblyPath = typeof(AssemblyToProcess.AllTypesObject).Assembly.Location;
+            _targetAssemblyPath = _sourceAssemblyPath.Replace(".dll", $".{_weaverOptions}.dll");
 
-            _sourceAssemblyPath = GetSourceAssemblyPath();
-            _targetAssemblyPath = CopyAssemblyToNextUniquePath(_sourceAssemblyPath);
+            if (File.Exists(_targetAssemblyPath))
```

Comment:
Yeah, I believe we can do without. It was left from an earlier iteration I had on this, but is now superfluous.